csharp/src/Ice/Properties.cs

```diff
@@ -4,6 +4,7 @@
 
 using Ice.Internal;
 using System.Globalization;
+using System.Net.NetworkInformation;
```

Comment:
Is this required?

Reply:
I don't think so. Not even sure how it got there to be honest.

Reply:
VSCode for sure! It some times add a using when you are writing a type , or using autocomplete.